docs/tools/index.md

```diff
@@ -4,7 +4,7 @@ title: fgsv tools
 
 # fgsv tools
 
-The following tools are available in fgsv version 20220218-d0eb31d.
+The following tools are available in fgsv version 20220323-9e90d4c.
```

Comment:
Should we just say "the current version of fgsv" since the commit hash will change on squash?

Reply:
This is all done automatically by `BuildToolDocs`, annoying but happy for someone to make it better.